Assignment2.yaml

```diff
@@ -0,0 +1,415 @@
+      url:
+        type: "string"
+      credited:
+        type: "string"
+      edited:
+        type: "string"
+  people:
+    type: "object"
+    required:
+    - "name"
+    properties:
+      name:
+        type: "string"
+        example: "Shiv"
+      birth_year:
+        type: "string"
+      eye_color:
+        type: "string"
+      gender:
+        type: "string"
+        example: "brown"
+      hair_color:
+        type: "string"
+        example: "black"
+      height:
+        type: "string"
+      mass:
+        type: "string"
+      skin_color:
+        type: "string"
+      homeworld:
+        type: "string"
+      films:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/planets/1/]"
+      species:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/species/1/]"
+      starships:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/starships/12/]"
+      vehicles:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/vehicles/14/]"
+      url:
+        type: "string"
+      created:
+        type: "string"
+      edited:
+        type: "string"
+  starship:
+    type: "object"
+    required:
+    - "name"
+    properties:
+      model:
+        type: "string"
+        example: "T-65 X-wing"
+      starship_Class:
+        type: "string"
+        example: "Starfighter"
+      name:
+        type: "string"
+        example: "Death Star"
+      Manufacturer:
+        type: "string"
+      Length:
+        type: "integer"
+        format: "int32"
+      cost_in_credits:
+        type: "string"
+      max_atmospheric_speed:
+        type: "string"
+      hyperdrive_rating:
+        type: "string"
+      MGLT:
+        type: "string"
+      cargo-capacity:
+        type: "string"
+      consumables:
+        type: "string"
+      films:
+        type: "string"
+      pilots:
+        type: "string"
+      url:
+        type: "string"
+      credited:
+        type: "string"
+      edited:
+        type: "string"
+  species:
+    type: "object"
+    required:
+    - "name"
+    properties:
+      name:
+        type: "string"
+        example: "mammal"
+      classification:
+        type: "string"
+        example: "senitent"
+      average_lifespan:
+        type: "string"
+      eye_color:
+        type: "string"
+      hair_color:
+        type: "string"
+      skin_color:
+        type: "string"
+      average_height:
+        type: "string"
+      mass:
+        type: "string"
+      language:
+        type: "string"
+      homeworld:
+        type: "string"
+      films:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/films/1/]"
+      species:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/species/1/]"
```

Comment:
In first place there is no such property in this model, but still if it was can you answer: Is this `array` or a `string`?
There is a difference between "[http://swapi.co/api/species/1/]" and [ "http://swapi.co/api/species/1/" ]